packages/app/source/client/pages/index/landing-page.html

```diff
@@ -2,23 +2,23 @@
 <div id="landing-page">
 
   <section class="donate">
-    <header>
+    <div class="headline">
       <h1>{{i18n "donate_things"}}</h1>
       <h2>{{i18n "voluntary_donors"}}</h2>
```

Comment:
Yeah we will refactor all of the translations at some point … but i think it "kind of makes sense" like that – those are the headlines of the landing page :wink: